### PR TITLE
Generate language from microgenerator

### PR DESCRIPTION
**Pull request by blowmage**

### CODE REVIEW

File: google-cloud-language/proto_docs/google/cloud/language/v1/language_service.rb [hunk 1]
```diff
@@ -12,16 +14,18 @@
...
   module Cloud
     module Language
       module V1
-        # ================================================================ #
+        # ################################################################ #
         #
         # Represents the input to API methods.
         # @!attribute [rw] type
-        #   @return [Google::Cloud::Language::V1::Document::Type]
+        #   @return [ENUM(Type)]
```
Comment (quartzmo) [inline google-cloud-language/proto_docs/google/cloud/language/v1/language_service.rb]: Are all of these `ENUM(*)` types correct?

File: google-cloud-language/.rubocop.yml [hunk 2]
```diff
@@ -3,11 +3,26 @@ inherit_gem:
 
 AllCops:
   Exclude:
+    - "acceptance/**/*"
```
Comment (blowmage) [inline google-cloud-language/.rubocop.yml]: Strange that it adds acceptance when there are no files being generated...

File: google-cloud-language/.rubocop.yml [hunk 3]
```diff
@@ -3,11 +3,26 @@ inherit_gem:
...
     - "google-cloud-language.gemspec"
-    - "lib/google/**/*"
+    - "lib/**/*_pb.rb"
+    - "proto_docs/**/*"
     - "Rakefile"
+    - "support/**/*"
```
Comment (blowmage) [inline google-cloud-language/.rubocop.yml]: This is used for doctest, and we don't have any plans to support this in generated gems afaik.

File: google-cloud-language/.yardopts [hunk 4]
```diff
@@ -1,11 +1,10 @@
 --no-private
---title=Cloud Natural Language API
+--title=Google Cloud Language
```
[the remark this answers is not in the document]
Comment (blowmage) [reply google-cloud-language/.yardopts]: Yes, this can be changed in the config YAML file.

File: google-cloud-language/Rakefile [hunk 5]
```diff
@@ -119,19 +135,13 @@ task :ci do
   header "BUILDING google-cloud-language"
   header "google-cloud-language rubocop", "*"
   Rake::Task[:rubocop].invoke
-  header "google-cloud-language doctest", "*"
-  Rake::Task[:doctest].invoke
+  header "google-cloud-language yard", "*"
```
Comment (blowmage) [inline google-cloud-language/Rakefile]: Swapped out doctest for YARD in the CI task. Really amazed that the docs build without warning!

File: google-cloud-language/lib/google/cloud/language/v1/language_service/client.rb [hunk 6]
```diff
@@ -0,0 +1,1005 @@
...
+            #
+            #   # Get sentiment for all sentences in the document
+            #   response.sentences.each do |sentence|
+            #     puts "Sentence text: #{sentence.text.content}"
+            #     puts "Sentence sentiment score: #{sentence.sentiment.score}"
+            #     puts "Sentence sentiment magnitude: #{sentence.sentiment.magnitude}"
+            #   end
+            #
+            #   # Get the language of the text, which will be the same as
+            #   the language specified in the request or, if not specified,
```
Comment (blowmage) [inline google-cloud-language/lib/google/cloud/language/v1/language_service/client.rb]: These additional comment lines are not commented out. We should probably split on `\n` to make sure each line gets a comment character in front of it.

**Comment by dazuma** [comment]
Closing this now. All the relevant comments have been addressed since this PR was opened.

**Pull request closed without merging**